src/main.rs

```diff
@@ -1,636 +1,10 @@
-                                    bits: 64,
-                                    value: self.get_int_op(self.vars.get(name).unwrap()),
-                                };
-                                self.vars.insert(
-                                    dest.clone(),
-                                    ConstantOperand(ConstantRef::new(constant)),
-                                );
-                            }
-                            _ => todo!(),
-                        };
-                    }
-                    _ => todo!(),
-                },
-                _ => todo!(),
-            },
-            ConstantOperand(_) => todo!(),
-            MetadataOperand => todo!(),
-        }
-    }
-
-    fn icmp(&mut self, pred: IntPredicate, op0: &Operand, op1: &Operand, dest: &name::Name) {
-        match op0 {
-            LocalOperand { name, ty } => match ty.as_ref() {
-                Type::IntegerType { bits: _ } => {
-                    let op0 = self.get_int_op(self.vars.get(name).unwrap());
-                    let op1 = self.get_int_op(op1);
-                    self.store_comp(pred, op0, op1, dest);
-                }
-                _ => todo!(),
-            },
-            ConstantOperand(con_op) => match con_op.as_ref() {
-                Constant::Int { .. } => todo!(),
-                Constant::Vector(_) => todo!(),
-                _ => todo!(),
-            },
-            MetadataOperand => todo!(),
-        }
-    }
-
-    fn store_comp(&mut self, pred: IntPredicate, op0: u64, op1: u64, dest: &name::Name) {
-        let is_true = match pred {
-            IntPredicate::EQ => op0 == op1,
-            IntPredicate::NE => op0 != op1,
-            IntPredicate::UGT => op0 > op1,
-            IntPredicate::UGE => op0 >= op1,
-            IntPredicate::ULT => op0 < op1,
-            IntPredicate::ULE => op0 <= op1,
-            IntPredicate::SGT => op0 > op1,
-            IntPredicate::SGE => op0 >= op1,
-            IntPredicate::SLT => op0 < op1,
-            IntPredicate::SLE => op0 <= op1,
-        };
-
-        let constant = Constant::Int {
-            bits: 8,
-            value: is_true.into(),
-        };
-        self.vars
-            .insert(dest.clone(), ConstantOperand(ConstantRef::new(constant)));
-    }
-
-    fn condbr(
-        &self,
-        cond: &Operand,
-        true_dest: &name::Name,
-        false_dest: &name::Name,
-    ) -> name::Name {
-        match cond {
-            LocalOperand { name, .. } => match self.get_int_op(self.vars.get(name).unwrap()) {
-                0 => false_dest.clone(),
-                1 => true_dest.clone(),
-                _ => unreachable!(),
-            },
-            ConstantOperand(_) => todo!(),
-            MetadataOperand => todo!(),
-        }
-    }
-}
+use llvm_ir::Module;
+use std::env;
 
 fn main() {
-    let mut bf = BFInterp::new();
+    let mut bf = llvm_ir_interpreter::BFInterp::new();
```

Comment:
We should also update `BFInterp`. And maybe we can shorten `llvm_ir_interpreter` too (maybe just `interp`)? Maybe the following structure would work:
```
main.rs
interp/
    mod.rs
```
Thoughts?